@Grunny seems like a linter has an issue with this PR
```
/app/models/wiki/file.js:
  34:7   error  Expected space(s) after "if"                                     keyword-spacing
  35:9   error  All 'var' declarations must be at the top of the function scope  vars-on-top
  35:9   error  Unexpected var, use let or const instead                         no-var
  37:9   error  Expected space(s) after "if"                                     keyword-spacing
  37:52  error  Missing space before opening brace                               space-before-blocks
  41:38  error  Multiple spaces found before 'redir'                             no-multi-spaces
```

--- a/app/models/wiki/file.js
+++ b/app/models/wiki/file.js
@@ -31,7 +31,7 @@ export default BaseModel.extend({
         fileUsageListSeeMoreUrl: get(data, 'nsSpecificContent.fileUsageListSeeMoreUrl'),
         fileThumbnail: media,
       };
-      if(!this.get('currentUser.isAuthenticated') && this.get('wikiVariables.enableFilePageRedirectsForAnons')) {
+      if (!this.get('currentUser.isAuthenticated') && this.get('wikiVariables.enableFilePageRedirectsForAnons')) {
         var redir = this.get('wikiVariables.articlePath') + this.get('wikiVariables.mainPageTitle');
 
         if(pageProperties.fileUsageList.length > 0){

